src/Server.hs

```diff
@@ -333,20 +332,19 @@ getAllFiles (ToodlesConfig ignoredPaths _) basePath =
 
     where
 
-    isValidFile :: FilePath -> Bool
-    isValidFile path = fileHasValidExtension && not ignoreFile
+    ignoreFile :: FilePath -> Bool
```

Comment:
since we're now using this to check directories, i think it would be good to rename this function to something like `ignorePath`. What do you think?